### ISSUE TITLE
New failing tests

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#3642]
Linux:
```
	295 - gmtmex/WL_example_3.sh (Failed)
	400 - grdimage/grdcyclic.sh (Failed)
```

macOS:
```
	400 - grdimage/grdcyclic.sh (Failed)
	677 - psscale/hardsoft.sh (Failed)
```

Windows:
```
	215 - api/apiuserdataset.sh (Failed)
	219 - api/apiuservec.sh (Failed)
	295 - gmtmex/WL_example_3.sh (Failed)
	387 - grdgdal/gdal_nn.sh (Failed)
	400 - grdimage/grdcyclic.sh (Failed)
	449 - grdvector/bothg.sh (Failed)
	459 - grdvector/vectors.sh (Failed)
	476 - greenspline/gspline_5.sh (Failed)
	488 - mapproject/oblmerc_down.sh (Failed)
	571 - psbasemap/map_JE.sh (Failed)
```

**Comment by PaulWessel** [comment]
On macos the only failing test are still imagepluscpt.sh and grdcyclic.png, both of which use @earth_relief_05m.  Those files have not changed on the server since June 1. The diff pngs are speckled, meaning very tiny changes. Would @seisman know after which commit things started to show this pattern?

**Comment by seisman** [comment]
Are these new failing tests? In yesterday nightly CI job, there are only two failing tests on macOS:
```
	387 - grdgdal/gdal_nn.sh (Failed)
	677 - psscale/hardsoft.sh (Failed)
```

**Comment by PaulWessel** [comment]
I shall do the simple experiment of removing my 05m files - perhaps that is the source.  hardsoft.sh aways passes here - will see what it does.

**Comment by seisman** [comment]
hardsoft.sh passes locally for me, but fails in CI jobs.
![image](https://user-images.githubusercontent.com/3974108/87496367-67728400-c621-11ea-8dc5-03def1faa03f.png)

**Comment by PaulWessel** [comment]
Interesting.  Removing my 05m files and redownloading them with gmt get and then rebuild by build and run tests gave exactly the same failures I ahd before. This is master branch.

**Comment by seisman** [comment]
The hardsoft.sh failure was fixed in #3885. Closing.
